jwql/website/apps/jwql/views.py

```diff
@@ -960,9 +965,24 @@ def view_image(request, inst, file_root, rewrite=False):
 
     form = get_anomaly_form(request, inst, file_root)
 
+    prop_id = file_root[2:7]
+
+    rootnames = get_rootnames_for_instrument_proposal(inst, prop_id)
+    file_root_list = defaultdict(list)
+
+    for root in rootnames:
+        try:
+            file_root_list[(filename_parser(root)['observation'])].append(root)
+        except KeyError:
+            pass
+
+    file_root_list = {key : sorted(file_root_list[key]) for key in sorted(file_root_list)}
```

Comment:
currently an initial sort.  Sorting to follow page sorting will be implemented with a future PR?

Reply:
So this was my first attempt at sorting and just figured I would do it alphabetically by default. Should we go ahead and remove it and implement it in another pr?

Reply:
Leaving it this way is a-ok.  Some sort of known sort is better than leaving it out.  Verifying that we weren't pursuing doing advanced sorting in this PR.